protected/humhub/components/ModuleManager.php

```diff
@@ -213,23 +216,91 @@ public function register($basePath, $config = null)
         Yii::$app->setModule($config['id'], $moduleConfig);
 
         // Register Event Handlers
-        if (isset($config['events'])) {
-            foreach ($config['events'] as $event) {
-                $eventClass = $event['class'] ?? $event[0];
-                $eventName = $event['event'] ?? $event[1];
-                $eventHandler = $event['callback'] ?? $event[2];
+        $events = $config['events'] ?? null;
+        $strict = $config['strict'] ?? false;
+        while (!empty($events)) {
+            $error = static function (string $message, bool $throw = false) use (&$config, $basePath) {
```

Comment:
Can we simply use a private method here? Or did I missed something?

Reply:
Ah, yes, see previous comment.